fix(clustered): adjust keycloak AppInstance config example (#5536)
influxdata/docs-v2: content/influxdb/clustered/install/configure-cluster.md

        jwksEndpoint: |-
          https://KEYCLOAK_HOST/auth/realms/KEYCLOAK_REALM/protocol/openid-connect/certs
        users:

>           # All fields are required but the firstName, lastName, and email can
>           # be arbitrary values -- only the `id` needs to match up with the
>           # identity provider user management system.

Reviewer: ```suggestion
          # All fields are required but `firstName`, `lastName`, and `email` can be
          # arbitrary values. However, `id` must match the user ID provided by Keycloak.
```
